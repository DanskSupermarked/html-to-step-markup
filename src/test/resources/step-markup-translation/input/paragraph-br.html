<p>God boxmadras med 265 bonnelfjedre pr. m2, i hårdheden medium, og 5 års garanti mod fjederbrud.<br /> <br /> Topmadras i 25 mm. purskum, med vaskbart betræk.                                       <br /> Silkedynen er af 100 % mulberrysilke, og er både antibakteriel og temperaturregulerende, så den er velegnet til brug hele året.<br /> Hovedpuden måler 60 x 63 cm og består af 150 gram mulberrysilke og en fiberkerne for optimal nakkestøtte med vår af lækkert bomuldssatin.  <br /> Inkl. hvidt kuvert- og boxmadraslagen i lækker blød bomuldskvalitet.</p><p>Komplet sæt bestående af:</p><p>HUMA Classic boxmadras 90 x 200 cm x 2 stk.     <br />Topmadras 180 x 200 cm        <br />Hvide træben 2 x 4 stk.        <br />Silkedyne- og pude, 2 sæt        <br />Kuvert- og boxmadraslagen i hvid  180 x 200 cm</p>
