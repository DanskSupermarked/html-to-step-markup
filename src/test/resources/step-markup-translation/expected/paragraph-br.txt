God boxmadras med 265 bonnelfjedre pr. m2, i hårdheden medium, og 5 års garanti mod fjederbrud.

Topmadras i 25 mm. purskum, med vaskbart betræk.
Silkedynen er af 100 % mulberrysilke, og er både antibakteriel og temperaturregulerende, så den er velegnet til brug hele året.
Hovedpuden måler 60 x 63 cm og består af 150 gram mulberrysilke og en fiberkerne for optimal nakkestøtte med vår af lækkert bomuldssatin.
Inkl. hvidt kuvert- og boxmadraslagen i lækker blød bomuldskvalitet.

Komplet sæt bestående af:

HUMA Classic boxmadras 90 x 200 cm x 2 stk.
Topmadras 180 x 200 cm
Hvide træben 2 x 4 stk.
Silkedyne- og pude, 2 sæt
Kuvert- og boxmadraslagen i hvid 180 x 200 cm
